src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -61,6 +63,12 @@
             "com/mathworks/ci/MatlabBuilder/runMatlabTests.m";
     private static final String AUTOMATIC_OPTION = "RunTestsAutomaticallyOption";
 
+    private static final String pdfReportStr     = "'PDFReport'";
+    private static final String tapResultsStr    = "'TapResults'";
```

Comment:
I agree that if options increases in the future we might need to look into a different strategy like config object or struct to pass these parameters else it might look crowded. 

I feel we can keep this as it is for now considering the readability point of view. Also, Nikhil mentioned that in future we will generate runMATLABTests.m file automatically and we can look into this as well at that point of time. I will create a separate task to address this.